parser.go

```diff
@@ -526,6 +526,20 @@ func (p *Parser) parseOption(s *parseState, name string, option *Option, canarg
 		}
 
 		err = option.Set(nil)
+	} else if option.isTerminated() {
+		var args []*string
+
+		if argument != nil {
+			return newErrorf(ErrInvalidTag, "terminated options' flag `%s' cannot use `='", option)
```

Comment:
Nit: we're talking about one option (possessive), so I think this should be `"terminated option's flag ..."`